frontend/Lighting.jsx

```diff
@@ -72,28 +72,32 @@ function Lighting() {
       colors={['#F9F6FF', '#CFDFF7']}
       style={{
         flex: 1,
-        padding: 10,
-        paddingTop: 80,
+        padding: 15,
         alignItems: 'center',
       }}
     >
-      <View style={{ flexDirection: 'row', justifyContent: 'space-between', marginBottom: 30 }}>
+      <View style={{
+        flexDirection: 'row',
+        marginBottom: 30,
+        marginTop: 10,
+        justifyContent: 'center',
+      }}
+      >
         <LightingTitle>Lighting</LightingTitle>
         <Switch
-          trackColor={{ false: '00FF00', true: 'FFFFFF' }}
```

Comment:
@vinaynagappala I thought you said that removing the trackColor would turn the switch into a dot on Android?

Reply:
Nah I think that was a rendering issue. As soon as I made the change, the switch would turn into a dot, but when the page was reloaded it worked fine.

Reply:
Ah ok. Sounds good then